src/backend/marsha/core/defaults.py

```diff
@@ -57,6 +61,11 @@
 
 LIVE_TYPE_CHOICES = ((RAW, _("raw")), (JITSI, _("jitsi")))
 
+JOIN_MODE_CHOICES = (
+    (OPEN, _("open")),
+    (CLOSED, _("closed")),
```

Comment:
Wouldn't it be clearer to call it something like 'automatic' and 'ask_for_approval' ?

Reply:
I agree, this work with 2 modes. Not when we will add a third one.

Reply:
yes, we can reword this.

WDYT about :
- ask_for_approval (current behavior)
- denied (no join allowed)

And for others upcoming modes :
- forced (users joins jitsi on load)
- automatic (no approval needed)
or
- automatic (users joins jitsi on load)
- no_approval (no approval needed)

Reply:
Looks good to me